### ISSUE TITLE
Missing Content Type with Solr server 5.1  & Magento 1.9.1.1

### ISSUE THREAD

**Issue by simbus82** [jeroenvermeulen/magento-solarium#90]
Connection to Solr	Success	 
Ping Solr	Success	 
Check Solr version	Success	 
ERROR	Solr HTTP error: OK (415) {"responseHeader":{"status":415,"QTime":1},"error":{"msg":"Missing ContentType","code":415}}

Hi have this problem after removing the default content type from curl.php file in solarium folder
https://github.com/jeroenvermeulen/magento-solarium/blob/master/lib/Solarium/Core/Client/Adapter/Curl.php#L156

This because i have more errore and self test give me a complete failure. After some readings i see that solution can help me.

But i can't have a working solr...

I have to edit something because i have solr 5.1? Thanks

**Comment by simbus82** [comment]
After reading something about 5.1 i see that there is some difference from solr 4.x for this magento module.
i have user in the line of code above this
$options['headers']['Content-Type'] = 'application/x-www-form-urlencoded';
Now i have the correct content type that Solr5.1 want.

Now i have this error
Solr HTTP error: OK (400) {"responseHeader":{"status":400,"QTime":1},"error":{"msg":"missing content stream","code":400}}

There is something wrong in my solr 5.1 :-(

**Comment by mr-jingles** [comment]
same here.. did you solve this issue?

**Comment by simbus82** [comment]
No :-( i don't know if jeroen is following this project.
Maybe if he reads some pull and some fork, he can add to the master so we
can try to work for solve this problem...


Simone Bussoni - Simbus82 Web & Tech
www.simbus82.net - Sviluppo Web, SEO & Web Marketing, Esperto in Joomla!
CMS.
Cell. 347 4483752 -  info@simbus82.net - simbus82@pec.it

Simbus82 Web & Tech è anche su LinkedIn
<http://it.linkedin.com/in/simonebussoni> - Google Plus
<https://plus.google.com/b/101720886213453609476/101720886213453609476> -
Facebook <https://www.facebook.com/Simbus82WebTech> - Twitter
<https://twitter.com/simbus82>

2015-07-01 9:00 GMT+02:00 mr-jingles <notifications@github.com>:

**Comment by toonvd** [comment]
I don't think we support version 5 and higher at the moment, some config etc has changed, also, our solarium library is not up to date.

**Comment by mr-jingles** [comment]
changed to Solr version 4.10.3 1644336 for now - everything works - thx!
